cypress/e2e/genreselect_spec.cy.js

```diff
@@ -0,0 +1,28 @@
+describe('GenreSelect spec', () => {
+    it('test GenreSelect component default', () => {
+      cy.visit('http://localhost:3000/')
+      cy.get('[data-testid="genreSelect"]').contains('all');
+      cy.get('[data-testid="genreSelect"]').contains('horror');
+      cy.get('[data-testid="genreSelect"]').contains('comedy');
+      cy.get('[data-testid="genreSelect"]').contains('fantasy');
+      cy.get('[data-testid="genreSelect"]').contains('romance');
+      cy.get('[data-testid="genreSelect"]').contains('adventure');
+      cy.get('[data-testid="genreSelect"]').contains('action');
+      cy.get('[data-testid="genreSelect"]').contains('drama');
```

Comment:
Some of these are not e2e tests but component tests.
In e2e tests the site interactions are checked, e.g. the whole flow of searching, sorting, filtering. In component tests it is checked whether the component is behaving correctly with different configurations, e.g. has the correct default values or the selected option is highlighted.